core/helpers.py

```diff
@@ -17,6 +18,18 @@ def build_social_link(template, request, title):
     )
 
 
+def cms_component_supports_activated_language(activated_language, languages):
+    for language in languages:
+        if language[0] == activated_language:
+            return True
+
+
+def cms_component_is_bidi(activated_language, languages):
+    if cms_component_supports_activated_language(
+     activated_language, languages):
```

Comment:
the indentation is a bit funky

Reply:
had to add an extra space as linter was complaining. is there a better way to do this?

Reply:
could do 
```
is_supported = cms_component_supports_activated_language(
    activated_language, languages
)
if is_supported:
    return translation.get_language_info(activated_language)['bidi']
```

buuut now that `cms_component_supports_activated_language` is one line we could simple do:
```
def cms_component_is_bidi(activated_language, languages):
    if any(code == activated_language for code, _ in languages):
        return translation.get_language_info(activated_language)['bidi']
```

Reply:
nvm i think i fixed it

Reply:
hmmm okay maybe i'll do that instead